doc/src/read_restart.rst

```diff
@@ -47,6 +47,12 @@ Restart files are saved in binary format to enable exact restarts,
 meaning that the trajectories of a restarted run will precisely match
 those produced by the original run had it continued on.
 
+Some information about a restart file can be gathered directly from the
```

Comment:
Don't see that this new command-line option is added to the Run_options.rst doc page ?